testgrid/config.yaml

```diff
@@ -3041,14 +3041,6 @@ dashboards:
     test_group_name: post-k8sio-cip
   - name: ci-k8sio-cip
     test_group_name: ci-k8sio-cip
-  - name: periodic-packages-pushed
-    test_group_name: periodic-kubernetes-e2e-packages-pushed
-    alert_options:
-      alert_mail_to_addresses: kubernetes-patch-release-team@googlegroups.com
-  - name: periodic-packages-install-deb
```

Comment:
while https://github.com/kubernetes/release/pull/820 enables verification for pushed packages, are we going to have tests for DEB and PRM installation? `periodic-packages-install-deb` did DEB only.

Reply:
@neolit123 -- yes, I've added a TODO for it. I'm just pulling the test now because it's been busted since 1.14.0 and there isn't a quick way to fix it.

Once I get the verifies to work on all packages and not just `kubeadm`, we'll start to get signal into what's not being pushed and work to fix it.

(Although, I already know the answer: updated versions of `kubernetes-cni` and `cri-tools`.)

Reply:
ok, thanks for the explanation.